sensors/metrics/metrics.go

```diff
@@ -0,0 +1,80 @@
+package metrics
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"go.uber.org/zap"
+
+	"github.com/argoproj/argo-events/common/logging"
+)
+
+const (
+	prefix = "argo_events"
+)
+
+// Metrics represents EventSource metrics information
+type Metrics struct {
```

Comment:
I'm sure workflows was correct to have a metrics service- when global variables are easier to use

Reply:
Merged them to one struct.